backend/cloudinary_config.py

```
@@ -0,0 +1,9 @@
+import cloudinary
+import cloudinary.uploader
+
+cloudinary.config(
+  cloud_name="dws8lpmua",
+  api_key="595136529363616",
+  api_secret="VwJl7HXcox_1U9qeeN0fVmoL_VY",
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "api_secret"

(S106)